### PR TITLE
Changed embeddings to numpy array

### PR DESCRIPTION
**Pull request by nirhutnik**

### CODE REVIEW

File: deepchecks/nlp/text_data.py [hunk 1]
```diff
@@ -91,13 +92,14 @@ class TextData:
...
         inferred. Only relevant if properties is not None.
-    embeddings : t.Optional[Union[pd.DataFrame, np.ndarray, str]] , default: None
-        The text embeddings for the samples. Embeddings must be given as either a pandas DataFrame or a path to a pandas
-        DataFrame compatible csv file, with the rows representing each sample and columns representing the different
-        embeddings dimensions. If None, no embeddings are set.
-        The number of rows in the embeddings DataFrame must be equal to the number of samples in the dataset, and the
-        order of the rows must be the same as the order of the samples in the dataset.
-        In order to calculate the default embeddings, use the `TextData.calculate_default_embeddings` function after
+    embeddings : t.Optional[Union[np.ndarray, pd.DataFrame, str]], default: None
+        The text embeddings for the samples. Embeddings must be given as either a numpy array (or a path to an .npy
```
Comment (noamzbr) [inline deepchecks/nlp/text_data.py:96]: you say "either" but don't note the pandas option later.

File: deepchecks/nlp/text_data.py [hunk 2]
```diff
@@ -91,13 +92,14 @@ class TextData:
...
         The names of the categorical properties columns. If None, categorical properties columns are automatically
         inferred. Only relevant if properties is not None.
-    embeddings : t.Optional[Union[pd.DataFrame, np.ndarray, str]] , default: None
-        The text embeddings for the samples. Embeddings must be given as either a pandas DataFrame or a path to a pandas
-        DataFrame compatible csv file, with the rows representing each sample and columns representing the different
-        embeddings dimensions. If None, no embeddings are set.
-        The number of rows in the embeddings DataFrame must be equal to the number of samples in the dataset, and the
-        order of the rows must be the same as the order of the samples in the dataset.
-        In order to calculate the default embeddings, use the `TextData.calculate_default_embeddings` function after
+    embeddings : t.Optional[Union[np.ndarray, pd.DataFrame, str]], default: None
```
Comment (noamzbr) [inline deepchecks/nlp/text_data.py:95]: is pandas also an option or not any more?

**Review by noamzbr — APPROVED** [review]

File: deepchecks/nlp/text_data.py (hunk 2 above)
Comment (nirhutnik) [reply deepchecks/nlp/text_data.py:95]: I thought about keeping it as a "secret" option. Not crashing the code, but not documented to keep it simple. WDYT?

Comment (noamzbr) [reply deepchecks/nlp/text_data.py:95]: mm why not mentioned that also a pandas df will be OK? it's not a lot of extra text

Comment (noamzbr) [reply deepchecks/nlp/text_data.py:95]: mmm ok nvm

**Pull request merged**